ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -199,9 +218,9 @@ class JITModBuilder {
   // The function inside which we build the IR for the trace.
   Function *JITFunc;
 
-  // Map JIT instruction to basic block index and instruction index of the
-  // corresponding AOT instruction.
-  std::map<Value *, std::tuple<size_t, size_t, Instruction *>> AOTMap;
+  // Map JIT instruction to basic block index, instruction index, function
+  // name, and stackframe of the corresponding AOT instruction.
```

Comment:
s/stackframe/stack frame index/ ?

Is a lower index an older frame?

Reply:
Yes.

Reply:
Fixed in 6d7cdbe